translations/README-uz.md

```diff
@@ -191,7 +191,7 @@ Ushbu loyihani boshlaganimda, men stack bilan heap-ni farqini bilmas edim,
 Big-O ni nimaligini bilmas edim, tree-lar yoki graph-lar haqida hech qanday tushunchaga 
 ega emas edim. Agar tartiblash algoritmini (sorting) yozishni so'rashganda, aniq qovun tushirgan 
 bo'lar edim. Umrimda ishlatgan data strukturalarni hammasini tayyor holda ishlatganman 
-(built-in library) va ularni asosida nimalar yotishini umuman bilmas edim. 
+(built-in library) va ularning asosida nimalar yotishini umuman bilmas edim. 
```

Comment:
qavslar kerak emas bu yerda

Reply:
qavsdan oldin o'zbekcha teksti bor buni: Umrimda ishlatgan data strukturalarni hammasini tayyor holda ishlatganman 
(built-in library) va ularning ....      adashmasam siz to'g'irlab bergan joy edi bu

Reply:
Aha tugri. Kurmabman :)